### ISSUE TITLE
Image tags are missing alternate texts

### ISSUE THREAD

**Issue by a18osken** [HGustavs/LenaSYS#9825]
Throughout the site in general <img> tags are missing the <alt> tag and text which is required for screenreaders to parse and is important for site accessability as well as SEO.

I think this issue should be split into several sub-issues concerning specific files.

Part of issue #9800
